pkg/controller/inferenceservice/resources/istio/virtualservice.go

```diff
@@ -210,8 +210,8 @@ func (r *VirtualServiceBuilder) CreateVirtualService(isvc *v1alpha2.InferenceSer
 		},
 		Route: predictRouteDestinations,
 		Retries: &istiov1alpha3.HTTPRetry{
-			Attempts:      3,
-			PerTryTimeout: RetryTimeout,
+			Attempts:      0,
+			PerTryTimeout: nil,
 		},
```

Comment:
Searched there's 2 definition of "retry" here,  one for predictor and another for explainer, is there a reason we only modify one and keep another?
BTW, there's test cases in controller_test.go and virtualservice_test.go, I'm afraid it will fail after the change.

Reply:
No reason, purely oversight from my part. I'll fix the tests as well, having some trouble getting them to run locally.

Reply:
Agree that this should be retried by end user, let's hold and see how knative's experiment is going if there is any regression.